community/kernel/src/main/java/org/neo4j/kernel/api/security/AuthSubject.java

```diff
@@ -41,6 +41,11 @@
     void setPassword( String password, boolean requirePasswordChange ) throws IOException, InvalidArgumentsException;
 
     /**
+     * Changes authentication status to SUCCESS if in PASSWORD_CHANGE_REQUIRED
+     */
+    void passwordChangeNoLongerRequired();
```

Comment:
I don't really understand what this method is supposed to do. Could the doc be made more clear?